Build/Build.cs

```diff
@@ -138,32 +145,37 @@ class Build : NukeBuild
                     cc => cc.SetProjectFile(Solution.Specs.Approval_Tests)));
         });
 
-    Target UnitTests => _ => _
+    Project[] Projects => new[]
+    {
+        Solution.Specs.FluentAssertions_Specs,
+        Solution.Specs.FluentAssertions_Equivalency_Specs
+    };
+
+    Target UnitTestsWin => _ => _
```

Comment:
Advantages to split the one big target is, that you can see what is skipped and why in the build summary

Now you have to actually look at the code to see what is skipped

If you agree with me: maybe this should be also applied to the `TestFrameworks` target..

Reply:
See here:
Windows:
<img width="817" alt="image" src="https://user-images.githubusercontent.com/49762557/211927819-e63e9d94-c6c1-4a19-9b4f-8db55a42a57c.png">


 Unix:
<img width="884" alt="image" src="https://user-images.githubusercontent.com/49762557/211927750-69526478-ce85-4251-805d-e57ae28c2ae6.png">